owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/PassCodeActivity.kt

```diff
@@ -204,6 +214,13 @@ class PassCodeActivity : BaseActivity() {
                 /// pass code accepted in request, user is allowed to access the app
                 passCodeError.visibility = View.INVISIBLE
                 hideSoftKeyboard()
+                if (passCodeViewModel.getPassCode()!!.length < passCodeViewModel.getNumberOfPassCodeDigits()) {
```

Comment:
If `getPassCode()` is null, the application will crash. Be careful with the operator `!!`

Reply:
Well, that's because the `getString` method of `SharedPreferencesProvider` returns `String?`, but semantically the idea is that execution only arrives here (`processFullPassCode` with `ACTION_CHECK`) when we have to check the already saved passcode, so it shouldn't return null. What do you think? Do you purpose another solution? Maybe checking `!= null` before the if?

Reply:
Yes, is a good solution.